File: 122 Best Time to Buy and Sell Stock II/122 Best Time to Buy and Sell Stock II.js
var maxProfit = function(prices){
    var tmp=0, sum=0;
    for(var i=0; i<prices.length; i++){
        if ( i!=0 && tmp < prices[i])
            sum += (prices[i] - tmp);
        tmp = prices[i];
    }
    return sum;
};
//Min:132ms

File: 122 Best Time to Buy and Sell Stock II/122 Best Time to Buy and Sell Stock II.py
class Solution(object):
    def maxProfit(self, prices):
        tmp = 0
        sum = 0
        for i in range(0, len(prices)):
            if i!=0 and tmp < prices[i]:
                sum += (prices[i] - tmp)
            tmp = prices[i]
        return sum
#Min:48ms